momentumnet/tests/test_transform.py

```diff
@@ -4,15 +4,15 @@
 import torch
 import torch.nn as nn
 
-from momentumnet import transform_to_momentumnet
+from momentumnet import transform_to_momentumnet, MomentumNet
 
-from torchvision.models import resnet18
+from torchvision.models import resnet101
 
 
 @pytest.mark.parametrize("use_backprop", [True, False])
 def test_resnet_vision(use_backprop):
     x = torch.randn((2, 3, 10, 10), requires_grad=True)
-    net = resnet18()
+    net = resnet101()
```

Comment:
better parametrize the arch in the `test_resnet_vision` function like `test_resnet_vision(use_backprop, depth="18")` and test the two depths.